cpp/src/arrow/flight/middleware.h

```diff
@@ -61,6 +60,10 @@ enum class FlightMethod : char {
   DoExchange = 9,
 };
 
+/// \brief Get a human-readable name for a Flight method.
+ARROW_FLIGHT_EXPORT
+std::string FlightMethodToString(FlightMethod method);
```

Comment:
We already use the `ToString` convention in other places, so perhaps:
```suggestion
std::string ToString(FlightMethod method);
```